nuklear.h

```diff
@@ -10687,7 +10802,7 @@ nk_convert(struct nk_context *ctx, struct nk_buffer *cmds,
                 t->string, t->length, t->height, t->foreground);
         } break;
         case NK_COMMAND_IMAGE: {
-            const struct nk_command_image *i = (const struct nk_command_image*)cmd;
+            const struct nk_command_image *i = (const struct nk_command_image*)cmd;;
```

Comment:
probably typo

Reply:
Good catch, always miss stuff like this.